help/deploy/releases/release-history.md

```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
+  * [#2904](https://github.com/datagrok-ai/public/issues/2904): When multi axis option is used. together with split, line chart is empty. 
+  
+### Bar chart
+* Improvement:
+  * [#2881](https://github.com/datagrok-ai/public/issues/2881): Bar chart: add an option to rotate / invert axes. 
+* Fixes:
+  * GROK-15887: Viewer window can't be resized in height when it's too high on the screen. 
+  * GROK-16234: Wrong label shortening. 
+  * GROK-16331: Molecules are rendered incorrectly on resizing bar chart. 
+  
+### Box plot
+* Improvements:
+  * [#2905](https://github.com/datagrok-ai/public/issues/2905): Add ''Size by" or "Shape by" or "color by'' options. 
+  * GROK-9748: Violin-state option. 
+* Fix:
+  * GROK-16244: Violin style: category bug. 
+ 
+### JS API
+* Improvements:
+  * [Compatibility tool](https://datagrok.ai/help/deploy/releases/compatibility/) for JS API changes.
+  * GROK-15771: RFC: Public API. 
+  * GROK-15827: Expose core viewer events to JS API. 
+  * GROK-16215: Force DG.FileInfo to have `name`. 
+* Fix:
+  * GROK-16228: `grok.functions.calls.list` returns incorrect Func's nqName functions. 
+
+### Data Access
+* Improvements:
+  * GROK-15413: Sharepoint data provider. 
+* Fixes:
+  * GROK-15979: Swagger: blank screen after loading swagger file. 
+  * GROK-16003: Queries: New Aggregation Query: PivotGridDbTable. 
+  * GROK-15967: Context panel: When you share the connection \- the information doesn`t update.
+  
+### Enhancements in packages
+* Improvements:
+  * GROK-15922: EDA: Linear Regression. 
+  * GROK-15759: PowerPack: 'Add new column': Formula edit improvements.
+  * GROK-11494: Widgets: events.
+  * [#2902](https://github.com/datagrok-ai/public/issues/2902): Create "Pivot table" tutorial part.
+* Fixes:
+  * GROK-15324: Widgets: Recent projects: context menu fixing. 
+  * [#2924](https://github.com/datagrok-ai/public/issues/2924): Curves: connectDots doesn't work if fit is turned off. 
 
+### [Bio](https://github.com/datagrok-ai/public/blob/master/packages/Bio/CHANGELOG.md)
+* Helm:
+  * GROK-15994: Color missing monomers. 
+  * GROK-15995: Colors for libraries monomers. 
+  * GROK-15796: Helm cell renderer fixes for conversion to Helm. 
+* Fixes:
+  * GROK-15525: MSA: add checking unsuitable data to avoid running MSA with them. 
+  * GROK-15996: Macromolecule cell renderer fails in long mode. 
+  * GROK-15798: Atomic Level fixing for units FASTA, UN alphabet.
+  * GROK-15793: Calculate Identity, Similarity throw Index out of bounds.
 
+### [Chem](https://github.com/datagrok-ai/public/blob/master/packages/Chem/CHANGELOG.md)
+* Fixes:
+  * GROK-15299: Functions can't be saved to a project with data sync. 
+  * GROK-15790: Transform to smiles doesn't change units. 
+ 
+### Scripting
+* Improvements and fixes:
+  * GROK-15424: Cascade parametrized inputs in scripts. 
```

Comment:
Would be nice to link to help articles that explain what the feature is (of course, not everything is documented yet - but for instance this feature should be)

Reply:
added the link:
https://datagrok.ai/help/datagrok/concepts/functions/func-params-annotation#function-inputs